packages/compass-e2e-tests/helpers/commands/create-index.ts

```diff
@@ -108,5 +108,11 @@ export async function createIndex(
   const indexComponent = await browser.$(indexComponentSelector);
   await indexComponent.waitForDisplayed();
 
+  // Wait for index to get ready before proceeding
+  await browser
```

Comment:
I'm wondering if this can pass before the in-progress index was ever added or without it ever getting added. Might be ideal to wait for the real index to appear in the table?

Reply:
Hmmm, maybe I'm misunderstanding, but I think the code above waits for the index row to appear in the table? Is that what you have in mind?

Reply:
Aah yes sorry I misread it. Then this is more like a "make sure nothing's left in progress once the new index appears" check which is perfect.